### DISCUSSION TITLE
Lowercase month name in date format

### DISCUSSION THREAD

**Discussion by ferenci-tamas** [quarto-dev/quarto-cli#11369] [Q&A] [answered]
### Description

Is it possible to have lowercase month name in date format? The [relevant part](https://quarto.org/docs/reference/dates.html#using-a-date-format) of the References only mentions `MMMM` which gives the month name with a capital first letter.

**Comment by mcanouil** [comment, marked as answer]
What's the use case?
The month capitalisation depends on the language.
If you don't set the language, then the default is taken and is English.
In English, months are capitalised.

<table>
<tr><th>Input</th><th>Output</th></tr>
<tr><td>

````qmd
---
format: html
date: today
date-format: DD MMMM YYYY
lang: fr
---
````

</td><td>

<img width="545" alt="image" src="https://github.com/user-attachments/assets/86fc2856-508a-4962-a472-d93ae84e6b93">

</td></tr>
</table>

**Reply by ferenci-tamas** [reply]
Aaaaargh, I simply forgot to set the language... thank you very much!
